packages/app/cypress/e2e/cypress-in-cypress.cy.ts

```diff
@@ -118,6 +118,12 @@ describe('Cypress in Cypress', { viewportWidth: 1500, defaultCommandTimeout: 100
       cy.get(`[data-cy="reporter-panel"]`).invoke('outerWidth').should('eq', 800)
       cy.percySnapshot('initial state')
 
+      cy.contains('[aria-controls=reporter-inline-specs-list]', 'Specs')
+      .click({ force: true })
```

Comment:
I'm not delighted about using`force: true`, but since the reporter itself is not the subject of the test and the element seems to like to detach from the DOM, this cuts down on flake. I see in some other tests we use the keyboard shortcut instead, presumably for the same reason. Would be good to address this better.

Reply:
I wonder if waiting for the inner spec run to complete before we perform our assertions could fix some of that flake... something like:

```js
  cy.findByLabelText('Stats').within(() => {
    cy.get('.passed .num', { timeout: 10000 }).should('have.text', '2')
  })
```